src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -229,6 +229,13 @@ public void launch(SlaveComputer computer, TaskListener listener) {
         }
     }
 
+    private WatchList setupWatches(KubernetesClient client, String namespace, String name, TaskListener taskListener) {
+        List<Watch> watches = new ArrayList<>();
+        watches.add(client.pods().inNamespace(namespace).withName(name).watch(watcher));
+        watches.add(client.events().inNamespace(namespace).withField("involvedObject.name", name).watch(new TaskListenerEventWatcher(name, taskListener)));
+        return new WatchList(watches);
+    }
+
```

Comment:
Unused?

```suggestion
```

Reply:
indeed, forgot to remove it before submitting the pr